releasenotes/notes/remove-deprecated-unroller-4693330708c681e0.yaml

```
@@ -0,0 +1,22 @@
+---
+upgrade:
+  - |
+    Removes the deprecated ``Unroller`` class in :mod:`qiskit.transpiler.passes.basis`.
+    This class was deprecated in Qiskit Terra 0.45 and has been replaced by 
```

Don't apply this suggestion as it's not worth wasting a CI round trip, we can fix this when we update the release notes for the final 1.0 release.

```suggestion
    This class was deprecated in Qiskit 0.45 and has been replaced by 
```